repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -135,12 +146,27 @@ class Interpreter(prompt0: Ref[String],
     s"Syntax Error: $msg\n$locationString"
   }
 
-  def processCorrectScript(parsedCode: Parsed.Success[Seq[Seq[String]]],
+  def processCorrectScript(rawParsedCode: Parsed.Success[Seq[(String, Seq[String])]],
                            evaluate: EvaluateCallback): Seq[ImportData] = {
 
+    var offset = 0
+    var parsedCode: mutable.ArrayBuffer[( String, Seq[String])] = mutable.ArrayBuffer()
```

Comment:
No need a type annotation here, and let's use `mutable.Buffer` instead of `mutable.ArrayBuffer` since that's what everyone else seems to use (even though it's the same)